kubernetes/cray-opa/templates/policies/spire.yaml

```diff
@@ -77,6 +77,9 @@ data:
           {"method": "GET",   "path": `^/apis/smd/hsm/v2/.*$`},
           {"method": "HEAD",  "path": `^/apis/smd/hsm/v2/.*$`},
 
+          #PCS -> Get power status. If desired over SMD power status
+          {"method": "GET", "path": `^/apis/power-control/v1/power-status.*$`},
```

Comment:
Can these calls be filtered by xname, etc?

Reply:
On the same note, whose the end user of these APIs? Is this being used by an application on the system?

Reply:
Yes, for example `GET /apis/power-control/v1/power-status?xname=x0c0s0b0n0&xname=x0c0s0b0n1'

Reply:
Thanks. This needs to be added to the chart/policies, is xname filtering is enabled.

Reply:
I'm not sure I understand. The above policy for SMD allows for `GET /apis/smd/hms/v2/State/Component` which gets the state of everything (no xname filtering). `GET /apis/power-control/v1/power-status` does the same. Does it really need additional filtering rules?